src/main/java/com/blackducksoftware/integration/jira/task/issue/handler/DataFormatHelper.java

```diff
@@ -165,6 +180,20 @@ private void appendVulnerabilitiesCommentText(final StringBuilder commentText, f
         commentText.append("\n");
     }
 
+    public String getBomLastUpdated(final ProjectVersionView projectVersion) {
+        try {
+            final VersionRiskProfileView riskProfile = blackDuckDataHelper.getResponse(projectVersion, ProjectVersionView.RISKPROFILE_LINK_RESPONSE);
+            final SimpleDateFormat dateFormat = new SimpleDateFormat();
+            return dateFormat.format(riskProfile.bomLastUpdatedAt);
+        } catch (final IntegrationException intException) {
+            logger.debug(String.format("Could not find the risk profile: %s", intException.getMessage()));
+        } catch (final NullPointerException npe) {
```

Comment:
Rather than catching an NPE, couldnt we check to see if the riskProfile is null?